hazelcast/src/main/resources/cluster.sh

```diff
@@ -1,22 +1,33 @@
-#!/bin/sh
+#!/bin/bash
 
 if [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
-   	echo "parameters: "
-   	echo "	-o, --operation	    : Executes cluster-wide operation. Operation can be 'get-state','change-state','shutdown','force-start','partial-start','get-cluster-version','change-cluster-version'."
-    echo "	-s, --state 	    : Updates state of the cluster to new state. New state can be 'active', 'frozen', 'passive', 'no_migration'."
-    echo "	-a, --address  	    : Defines which ip address hazelcast is running. Default value is '127.0.0.1'."
-   	echo "	-p, --port  	    : Defines which port hazelcast is running. Default value is '5701'."
-   	echo "	-g, --groupname     : Defines groupname of the cluster. Default value is 'dev'."
-   	echo "	-P, --password      : Defines password of the cluster. Default value is 'dev-pass'."
-   	echo "	-v, --version       : Defines the cluster version to change to. To be used in conjunction with '-o change-cluster-version'."
-   	exit 0
+    echo "parameters: "
+    echo "  -o, --operation  : Executes cluster-wide operation. Operation can be 'get-state','change-state','shutdown','force-start','partial-start','get-cluster-version','change-cluster-version'."
+    echo "  -s, --state      : Updates state of the cluster to new state. New state can be 'active', 'frozen', 'passive', 'no_migration'."
+    echo "  -a, --address    : Defines which ip address hazelcast is running. Default value is '127.0.0.1'."
+    echo "  -p, --port       : Defines which port hazelcast is running. Default value is '5701'."
+    echo "  -g, --groupname  : Defines groupname of the cluster. Default value is 'dev'."
+    echo "  -P, --password   : Defines password of the cluster. Default value is 'dev-pass'."
+    echo "  -v, --version    : Defines the cluster version to change to. To be used in conjunction with '-o change-cluster-version'."
+    echo "  -d, --debug      : Prints curl error output."
```

Comment:
This is probably intentional, but just to be sure: is it important that we mention curl here?